enterprise/api_client/enterprise_catalog.py

```diff
@@ -135,7 +135,7 @@ def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
                 )
                 response = {}
```

Comment:
The "empty" response object, in the case of an exception, should include the `results` key as an empty list if we're always going to try iterating through the results in the `for...in` loop below.

Alternatively, perhaps we only try to iterate through the results if an exception does not occur, by moving the `for...in` loop within the try block above.